src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -131,12 +132,28 @@ protected function isPropertyMapped(string $property, string $resourceClass, boo
      * Determines whether the given property is nested.
      *
      * @param string $property
+     * @param string $resourceClass
+     *
+     * @return bool
+     */
+    protected function isPropertyNested(string $property, string $resourceClass): bool
+    {
+        return false !== strpos($property, '.') && false === $this->isPropertyEmbedded($property, $resourceClass);
+    }
+
+    /**
+     * Determines whether the given property is embedded.
+     *
+     * @param string $property
+     * @param string $resourceClass
      *
      * @return bool
      */
-    protected function isPropertyNested(string $property): bool
+    protected function isPropertyEmbedded(string $property, string $resourceClass): bool
     {
-        return false !== strpos($property, '.');
+        $metadata = $this->getClassMetadata($resourceClass);
+
+        return $metadata->hasField($property);
```

Comment:
@teohhanhui , the metadata refers embed field like with the "." hasField("embed.field") works only. Do you want I override the Symfony method ?
thanks for your feedback